### DISCUSSION TITLE
Only showing errors/warning from linters on modified lines

### DISCUSSION THREAD

**Discussion by mflova** [w0rp/ale#4015] [Ideas]
The typical developper workflow requires to create Pull Requests to apply the necessary changes to a repository. Since I usually like working with stricter linter options than the ones requested, my ALE linters display many different warnings and errors that were previously introduced in the file. In my case, I would only need to see those that correspond to my current implementation. Would it be possible to hide those warnings that do not belong to my modified/added code lines? 

About a possible implementation:
I do not mind if it is not clean.
I have seen that for those who use gitgutter, you can select the base of the diff with the following variable:
`let g:gitgutter_diff_base = 'master'`
Is there an easy way to hide those ALE warnings/errors whose line contains a + or ~ from the gitgutter output?

**Comment by hsanson** [comment]
Possibly related: https://github.com/dense-analysis/ale/issues/850

From that issue I get this is not an easy thing to do and is highly dependent on the linter/fixer used.
